.travis.yml

```diff
@@ -7,17 +7,10 @@ python:
 - '3.10'
 - '3.11'
 env:
-- PINT_VERSION=0.18
-- PINT_VERSION=0.21
-jobs:
-  exclude:
-    - python: 3.7
-      env: PINT_VERSION=0.21
+  - UPGRADES="pint --upgrade pandas --upgrade"
```

Comment:
`--upgrade` assumes the package is already installed. I don't think you need to flag those packages for upgrade and the `--upgrade` flag should come after install but before the package and would only be needed once IIRC. 

Example call: https://pip.pypa.io/en/stable/installation/#upgrading-pip